cpu/driver/runtest.sh

```diff
@@ -25,6 +25,25 @@ TMPDIR=/var/tmp/$(date +"%Y%m%d%H%M%S")
 
 source $CDIR/../../cpu/common/libutil.sh
 
+NON_PSTATE_PROCESSORS="26 31 46"
+check_pstate_support()
+{
+    typeset model=$(lscpu | grep Model: | awk '{print $2}')
+    if [ -z "$model" ]; then
+	cki_log "unable to determine cpu model"
+	return 1
+    fi
+
+    for m in $NON_PSTATE_PROCESSORS; do
+	if [ $m -eq $model ]; then
+	    cki_log "model: $model - does not support intel_pstate"
+	    return 1
```

Comment:
This sounds like it's unsupported, should it return a skip ?

Reply:
This is in check_pstate_support() which returns a 0 on success (meaning pstate is supported) and 1 on failure (meaning it is not).  So this is correct.